### ISSUE TITLE
get clientPropertiesData returns null in Sonoma 14.6

### ISSUE THREAD

**Issue by nyxaria** [kirtan-shah/nowplaying-cli#25]
As per title, running with the latest version 1.2.1 on Sonoma 14.6 gives null for the following:

```
Projects % nowplaying-cli get clientPropertiesData
> null
```

Before it would give encoded information about the application used to play the media. Is there any way to get this back?

**Comment by nyxaria** [comment]
I think I tracked it down - when I try running the bit of code that gets the `clientPropertiesData` in `Xcode`, it tells me that I need entitlement `com.apple.mediaremote.external-artwork-validation` and throws the error `null` which corresponds to the output above
<img width="857" alt="image" src="https://github.com/user-attachments/assets/f74566ff-b375-43d2-9238-14c7e9415132">

Would it be possible for nowplaying-cli to get this `com.apple.mediaremote.external-artwork-validation`? I tried to do this but failed - the approach I took is this one: https://developer.apple.com/documentation/xcode/signing-a-daemon-with-a-restricted-entitlement (except adding our entitlement too)

But it seems we need to pay for the apple development license to sign it? I got stuck at that point - if anyone knows better please do let me know :)

**Comment by nyxaria** [comment]
@kirtan-shah if you have any idea of this signing, could you please chime in?

**Comment by AzimovParviz** [comment]
have you figured it out yet? Trying to understand what output you even get from this property. Would it be possible to get the name of the application that is the source of the media playing for example? I see in README that it's some kind of raw data which seems to be supported by the source code.

**Comment by nyxaria** [comment]
> have you figured it out yet? Trying to understand what output you even get from this property. Would it be possible to get the name of the application that is the source of the media playing for example? I see in README that it's some kind of raw data which seems to be supported by the source code.

see https://github.com/kirtan-shah/nowplaying-cli/issues/7

**Comment by AzimovParviz** [comment]
> does this mean that for you you're able to get this property? For me it just returns null

no, the one time I tried it did not work. I ended up writing the functionality myself for my app
